src/shared/components/palette/fixed-list/read-fixed-list-field.component.ts

```diff
@@ -3,6 +3,23 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: 'ccd-read-fixed-list-field',
-  template: '<span class="text-16">{{caseField.value | ccdFixedList:caseField.field_type.fixed_list_items}}</span>',
+  template: '<span class="text-16">{{getValue() | ccdFixedList: getLists()}}</span>',
 })
-export class ReadFixedListFieldComponent extends AbstractFieldReadComponent {}
+export class ReadFixedListFieldComponent extends AbstractFieldReadComponent {
+
+  getValue() {
+    if (this.caseField.field_type.type === 'DynamicList') {
+      return this.caseField.value ? this.caseField.value.value.code : '';
+    } else {
+      return this.caseField.value;
+    }
+  }
+
+  getLists() {
```

Comment:
same as above please, let's move this in an accessor method in `CaseField`

Reply:
Will address accessor method in fixed-list